src/sfx.asm

```diff
@@ -0,0 +1,192 @@
+;
+; Constants for audio effects
+
+; Values for hMusicTrack
+ 
+MUSIC_NONE                                      equ $00
```

Comment:
It seems all those constants below are duplicated from the ones declared in `constants/sfx.asm`.

I'd rather have the constants only in a single place. Is it possible for you to delete those duplicate declarations in `constants/sfx.asm`?

Reply:
I think I accidentally upladed it to the wrong folder, let me check.

Reply:
constants/sfx.asm has been removed.